weblab/templates/datasets/includes/upload.html

```diff
@@ -8,7 +8,8 @@
     Drop Files Here<br /> <a>Open Dialog</a>
     <input type="file" name="upload" id="fileupload"
         data-url="{% url 'datasets:upload_file' dataset.id %}"
-        data-form-data='{"csrfmiddlewaretoken": "{{ csrf_token }}"}'>
+        data-form-data='{"csrfmiddlewaretoken": "{{ csrf_token }}"}'
+        required-file-type=".csv">
```

Comment:
I think this should be `data-required-file-type` to confirm with HTML spec/typical usage? And then use `.data` rather than `.attr` to retrieve the value?